src/base/solvers/configEnvVars.m

```diff
@@ -186,7 +186,8 @@ function configEnvVars(printLevel)
 
         % check for 32-bit
         if ~isempty(strfind(computer('arch'), '64'))
-            subDir = [filesep, osPath, filesep, 'matlab'];
+            subDir = [filesep, 'matlab'];
+            %subDir = [filesep, osPath, filesep, 'matlab'];
         end
```

Comment:
That old code does not work with my installation of gurobi. @laurentheirendt check how you were doing it before because the old path I think was wrong. The init functions, and their called subfunctions should have inputs and outputs commented. It was very hard to narrow down the problem. Also, gurobi should not be removed from the path after a different solver is chosen. I commented out that code also.